tests/test_sqlalchemy_athena.py

```diff
@@ -533,3 +533,49 @@ def test_conn_str_poll_interval(self, engine, conn):
     @with_engine(kill_on_interrupt="false")
     def test_conn_str_kill_on_interrupt(self, engine, conn):
         self.assertFalse(conn.connection.kill_on_interrupt)
+
+    @with_engine()
+    def test_create_table(self, engine, conn):
+        # Given
+        table_name = 'manually_defined_table'
+        table = Table(table_name,
+                      MetaData(),
+                      Column('c', String(10)),
+                      schema=SCHEMA,
+                      awsathena_location=f'{ENV.s3_staging_dir}/{SCHEMA}/{table_name}',
+                      )
+        insp = sqlalchemy.inspect(engine)
+
+        # When
+        table.create(bind=conn)
+
+        # Then
+        self.assertTrue(insp.has_table(table_name, schema=SCHEMA))
+
+        # Teardown
+        conn.execute('DROP TABLE IF EXISTS {}'.format(table.name))
+
+    def test_create_table_location(self):
+        """Ensure the location is properly inserted when the `awsathena_location` is used
+        and that a trailing slash is appended if missing.
+        """
+        # Given
+        expected_location = "s3://my-bucket/"
+        for location in ["s3://my-bucket", expected_location]:
+            table = Table(
+                'table_name',
+                MetaData(),
+                Column("column_name", String),
+                schema=SCHEMA,
+                awsathena_location=location,
+            )
+            dialect = AthenaDialect()
+            location_pattern = re.compile(r"(?:LOCATION ')([^']+)(?:')")
+
+            # When
+            statement = CreateTable(table).compile(dialect=dialect)
+
+            # Then
+            assert statement is not None
+            assert location_pattern.findall(str(statement))[0] == expected_location
+            assert len(location_pattern.findall(str(statement))) == 1
```

Comment:
```suggestion
        dialect = AthenaDialect()
        table = Table(
            "test_create_table",
            MetaData(),
            Column("column_name", String),
            schema="test_schema",
            awsathena_location="s3://path/to/test_schema/test_create_table",
            awsathena_compression="SNAPPY",
        )
        actual = CreateTable(table).compile(dialect=dialect)
        # If there is no `/` at the end of the `awsathena_location`, it will be appended.
        self.assertEqual(
            str(actual),
            textwrap.dedent(
                """
                CREATE EXTERNAL TABLE test_schema.test_create_table (
                \tcolumn_name VARCHAR
                )
                STORED AS PARQUET
                LOCATION 's3://path/to/test_schema/test_create_table/'
                TBLPROPERTIES ('parquet.compress'='SNAPPY')\n\n\n
                """
            ),
        )
```